src/renderers/testing/ReactTestRendererFiberEntry.js

```diff
@@ -611,6 +605,21 @@ var ReactTestRendererFiber = {
         return TestRenderer.getPublicRootInstance(root);
       },
     };
+
+    Object.defineProperties(entry, {
+      root: {
+        get: function() {
+          if (root === null || root.current.child === null) {
+            throw new Error("Can't access .root on unmounted test renderer");
+          }
+          return wrapFiber(root.current.child);
+        },
+        enumerable: true,
+        configurable: true,
```

Comment:
I was copying what the babel transform would do - seems like a reasonable default. They would both default to `false` otherwise.

Since it's not `_root`, it seems like it should be enumerable and configurable.